### PR TITLE
tiflash: update config description of max_memory_usage_for_all_queries

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

update config description of max_memory_usage_for_all_queries

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

code: https://github.com/pingcap/tiflash/pull/6607

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by windtalker — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -157,11 +157,19 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
-    ## 单次 coprocessor 查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
+    ## 单次查询过程中，节点对中间数据的内存限制
+    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32GB 的内存限制；其中 0 表示无限制
+    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
+    ## 默认值为 0，表示不限制
+    ## 当查询试图申请超过限制的内存时，会终止执行并且报错
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:164]:
```suggestion
    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GB 的内存限制；其中 0 表示无限制
    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
    ## 默认值为 0，表示不限制
    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -157,11 +157,19 @@ delta_index_cache_size = 0
...
     max_memory_usage = 0
 
-    ## 所有查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
-    max_memory_usage_for_all_queries = 0
+    ## 所有查询过程中，节点对中间数据的内存限制 
+    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32GB 的内存限制；其中 0 表示无限制
+    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
+    ## 默认值为 0.8，表示总内存的 80%
+    ## 当查询试图申请超过限制的内存时，会终止执行并且报错
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:171]:
```suggestion
    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GB 的内存限制；其中 0 表示无限制
    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
    ## 默认值为 0.8，表示总内存的 80%
    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -157,11 +157,19 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
-    ## 单次 coprocessor 查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
+    ## 单次查询过程中，节点对中间数据的内存限制
+    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GB 的内存限制；其中 0 表示无限制
+    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
+    ## 默认值为 0，表示不限制
+    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
```
Comment (Oreoxmt) [inline tiflash/tiflash-configuration.md:164]:
```suggestion
    ## 单次查询过程中，节点对中间数据的内存限制
    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GiB 的内存限制，其中 0 表示无限制
    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%，其中 0.0 表示无限制
    ## 默认值为 0，表示不限制
    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -157,11 +157,19 @@ delta_index_cache_size = 0
...
     max_memory_usage = 0
 
-    ## 所有查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
-    max_memory_usage_for_all_queries = 0
+    ## 所有查询过程中，节点对中间数据的内存限制 
+    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GB 的内存限制；其中 0 表示无限制
+    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%；其中 0.0 表示无限制
+    ## 默认值为 0.8，表示总内存的 80%
+    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
+    max_memory_usage_for_all_queries = 0.8
```
Comment (Oreoxmt) [inline tiflash/tiflash-configuration.md:172]:
```suggestion
    ## 所有查询过程中，节点对中间数据的内存限制 
    ## 设置为整数时，单位为 byte，比如 34359738368 表示 32 GiB 的内存限制，其中 0 表示无限制
    ## 设置为 [0.0, 1.0) 之间的浮点数时，指节点总内存的比值，比如 0.8 表示总内存的 80%，其中 0.0 表示无限制
    ## 默认值为 0.8，表示总内存的 80%
    ## 当查询试图申请超过限制的内存时，查询终止执行并且报错
    max_memory_usage_for_all_queries = 0.8
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**